Initialization of RemotePSMP causes NullPointerException

**App version:** 1.6.6 BETA (if I understand this bug correctly, it also occurs in earlier versions)

**Android version**: 9.0 (should occur in earlier versions)

**Steps to reproduce**:

1. Enable chromecast
1. Chromecast is playing when app if opened

**Stacktrace/Logcat**: 
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.handleCreateService (ActivityThread.java:3544)
  at android.app.ActivityThread.access$1300 (ActivityThread.java:199)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1666)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:193)
  at android.app.ActivityThread.main (ActivityThread.java:6669)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:858)
Caused by: java.lang.NullPointerException: 
  at de.danoeh.antennapod.core.service.playback.PlaybackService$2.statusChanged (PlaybackService.java:663)
  at de.danoeh.antennapod.core.service.playback.PlaybackServiceMediaPlayer.setPlayerStatus (PlaybackServiceMediaPlayer.java:326)
  at de.danoeh.antennapod.core.service.playback.RemotePSMP.onRemoteMediaPlayerStatusUpdated (RemotePSMP.java:221)
  at de.danoeh.antennapod.core.service.playback.RemotePSMP.<init> (RemotePSMP.java:68)
  at de.danoeh.antennapod.core.service.playback.PlaybackServiceFlavorHelper.onCastAppConnected (PlaybackServiceFlavorHelper.java:178)
  at de.danoeh.antennapod.core.service.playback.PlaybackService.onCreate (PlaybackService.java:4068)
  at android.app.ActivityThread.handleCreateService (ActivityThread.java:3532)```

--- a/core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java
+++ b/core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java
@@ -660,7 +660,12 @@ public class PlaybackService extends MediaBrowserServiceCompat {
     private final PlaybackServiceMediaPlayer.PSMPCallback mediaPlayerCallback = new PlaybackServiceMediaPlayer.PSMPCallback() {
         @Override
         public void statusChanged(PlaybackServiceMediaPlayer.PSMPInfo newInfo) {
-            currentMediaType = mediaPlayer.getCurrentMediaType();
+            if (mediaPlayer != null) {
+                currentMediaType = mediaPlayer.getCurrentMediaType();
+            } else {
+                currentMediaType = MediaType.UNKNOWN;
+            }
+
             updateMediaSession(newInfo.playerStatus);
             switch (newInfo.playerStatus) {
                 case INITIALIZED:

